tests/baselines/reference/contextualTypeAny.errors.txt

```diff
@@ -0,0 +1,15 @@
+tests/cases/compiler/contextualTypeAny.ts(3,5): error TS2322: Type '{ p: string; q: any; }' is not assignable to type '{ [s: string]: number; }'.
+  Property 'p' is incompatible with index signature.

```

Comment:
Right, but the whole point of our elaboration scheme is to not try to show everything on one line. The respective types of `p` and the index signature follow in the next elaboration. It's completely analogous to how we report a mismatch between two properties.